File: TripKitAndroid/src/test/java/com/skedgo/android/tripkit/TestRunner.java
package com.skedgo.android.tripkit;

import org.junit.runners.model.InitializationError;
import org.robolectric.RobolectricTestRunner;

public class TestRunner extends RobolectricTestRunner {
  public TestRunner(Class<?> klass) throws InitializationError {
    super(klass);
  }
}

File: trip-kit-booking-ui/src/test/java/com/skedgo/android/tripkit/booking/ui/TestRunner.kt
package com.skedgo.android.tripkit.booking.ui

import org.junit.runners.model.InitializationError
import org.robolectric.RobolectricTestRunner

class TestRunner @Throws(InitializationError::class) constructor(
    klass: Class<*>
) : RobolectricTestRunner(klass)
